Source/DafnyCore/DafnyOptions.cs

```diff
@@ -1332,9 +1332,8 @@ the program.
 
 /extractCounterexample
     If verification fails, report a detailed counterexample for the
-    first failing assertion. Requires specifying the /mv option as well
-    as /proverOpt:O:model_compress=false and
-    /proverOpt:O:model.completion=true.
+    first failing assertion. Requires specifying the /mv:<file> option as well
+    as /proverOpt:O:model_compress=false (for z3 version < 4.8.7) or /proverOpt:O:model.compact=false (for z3 version >= 4.8.7), and /proverOpt:O:model.completion=true.
```

Comment:
```suggestion
    as /proverOpt:O:model_compress=false (for z3 version < 4.8.7) or
    /proverOpt:O:model.compact=false (for z3 version >= 4.8.7), and
    /proverOpt:O:model.completion=true.
```

Reply:
I recently reformatted all of the option descriptions so they wouldn't wrap on a standard 80-column terminal, and I'd love it if we could keep them that way.